packages/core/src/internal-components/overlay/overlay.element.ts

```diff
@@ -101,7 +101,8 @@ export class CdsInternalOverlay extends CdsBaseFocusTrap implements Animatable {
   // renderRoot needs delegatesFocus so that focus can cross the shadowDOM
   // inside an element with aria-modal set to true
   /** @private */
-  static get shadowRootOptions() {
+  static get shadowRootOptions(): any {
+    // any is used until TS 4.4.x adopted through other @cds/* libraries. Can be removed in 6.0
```

Comment:
I set these to any so the 5.x @cds/* libraries continue to work. Once we branch for 6.x we can remove the `any` and use the update types that TS provides.

Reply:
Zero concerns about leaving `any` here as long as we want.